pipeline/qgis_export/geojson.py

```diff
@@ -0,0 +1,28 @@
+from pipeline.preprocessing.compute_features.feature import Feature
+
+
+import pandas as pd
+
+
+def annotate_geojson_with_speedlimit(
+    geojson: dict, df_with_speedlimit: pd.DataFrame
+) -> None:
+    """Annotates a geojson dict with predicted speedlimits from a dataframe.
+    If there is no predicted value for a segment, the default will be 'na'.
+    Args:
+        geojson (dict): geojson as a dictionary
+        df_with_speedlimit (pd.DataFrame): dataframe with speedlimits
+    """
+    # index using osm_id
+    attribute_name = "speed limit"
+    df_with_speedlimit["index"] = df_with_speedlimit[Feature.OSM_ID.value]
+    df_with_speedlimit = df_with_speedlimit.set_index("index")
+
+    for segment in geojson["features"]:
```

Comment:
Det ligner ikke at den dictionary vi får fra osm_geometry.py har den her header? Eller er det en anden dictionary der bruges?

Reply:
It is fine